### PR TITLE
fix: Update the way build info is fetched from GitHub

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Proposed changes

We have changed the way our monorepo is tagged and thus it's necessary to update the build info retrieval logic that uses GitHub REST API. Previously the /status call was returning some random stuff

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/appium/lib/appium.js [hunk 1]
```diff
@@ -108,10 +108,13 @@ class AppiumDriver extends DriverCore {
 
     // allow this to happen in the background, so no `await`
     // catch this to avoid an unhandled rejection
```
Comment (jlipps) [inline packages/appium/lib/appium.js:110]: `make sure we catch any possible errors to avoid unhandled rejections`

**Review by jlipps — APPROVED** [review]

File: packages/appium/lib/appium.js (hunk 1 above)
Comment (mykola-mokhnach) [reply packages/appium/lib/appium.js:110]: not sure what the issue is?

Comment (jlipps) [reply packages/appium/lib/appium.js:110]: we are not catching "this" anymore. now we are using a catch inside the IIFE async function. i was giving a suggestion for rewording the comment to apply more accurately.

Comment (mykola-mokhnach) [reply packages/appium/lib/appium.js]: nvm, updated the comment

**Pull request merged**